seahorn/storage/ipc/jobs/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+list(APPEND JOB_LISTS storage_ipc_indirect_handlers 
+storage_ipc_msg_buffer storage_ipc_port_create_destroy)
```

Comment:
this goes against typical cmake convention. I would create a macro `add_job` and use it to create build recipe for every job. Better yet, put individual cmake files in every job directory